docs/data/toolpad/core/components/page-container/PageContent.js

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
 import Typography from '@mui/material/Typography';
-import Grid from '@mui/material/Unstable_Grid2';
+import Grid from '@mui/material/Grid2';
```

Comment:
🤔  We may want to keep backwards compatibility with v5.

@mnajdova Will the `/Unstable_Grid2` path be maintained during v6?